README.md

```diff
@@ -1036,24 +1037,23 @@ You may wish to get a brief summary of failures with commands such as:
 
 ```bash
 $ cd <exiv2dir>/build
-$ make python_tests 2>&1 | grep FAIL
+$ ctest -R bugfixes --verbose 2>&1 | grep FAIL
 ```
 
 [TOC](#TOC)
 <div id="4-5">
-
 ### 4.5 Test Summary
 
-| *Tests*      | Unix Style Platforms _(bash)_      | Visual Studio _(cmd.exe)_             |
+| *Tests*      | Execute using ctest      | Execute using cmake      |
 |:--           |:---                                |:--                                    |
-|              | $ cd \<exiv2dir\>/build            |  \> cd \<exiv2dir\>/build             |
-| tests        | $ make tests                       | \> cmake --build . --config Release --target tests |
-| bash_tests   | $ make bash_tests                  | \> cmake --build . --config Release --target bash_tests |
-| python_tests | $ make python_tests                | \> cmake --build . --config Release --target python_tests |
-| unit_test    | $ make unit_test                   | \> cmake --build . --config Release --target unit_test |
-| version_test | $ make version_test                | \> cmake --build . --config Release --target version_test |
-
-The name **bash_tests** is historical.  They are implemented in python.
+|              | $ cd \<exiv2dir\>/build            | \> cd \<exiv2dir\>/build              |
+| test         | $ ctest                            | \> cmake --build . --config Release --target test         |
```

Comment:
The `--config Release` is now redundant everywhere since we made it default.

Reply:
I need to carefully consider and document `cmake --config option`.